google/cloud/opentelemetry/README.md

```diff
@@ -1,4 +1,8 @@
-# C++ OpenTelemetry Exporters for Google Cloud Platform services.
+# Google Cloud C++ OpenTelemetry Exporters
+
+This directory contains the [OpenTelemetry] [trace exporter][trace-exporter] for
+[Cloud Trace][cloud-trace]. It also contains helpers to install the Cloud Trace
```

Comment:
nit: I think this paragraph assumes the reader knows what a trace exporter is, and why one may want one of them.  Maybe say:

```
This directory contains components to export [traces] captured with OpenTelemetry to [Cloud Trace].
Traces are a form of instrumentation where calls across multiple distributed systems can be collected
in a centralized view, showing the sequence of calls and their latency. [Cloud Trace] can provide such
views of your system calls.

OpenTelemetry offers APIs to capture traces. The C++ client libraries are already instrumented, you can use this instrumentation with minimal changes to your application. It is not necessary to instrument your application too.
```

Feel free to massively edit or expand that.  Just assume the reader is learning about tracing for the first time.